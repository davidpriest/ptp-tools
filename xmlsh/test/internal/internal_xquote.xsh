# test of xquote

a=<[ <foo>bar</foo> ]>
# a is a element
xtype $a

set +indent
# $b is a string
xquote {$a}>{b}
xtype $b

# Test results within elements
xecho <[ <test>{$a}</test> ]>
xecho <[ <test>{$b}</test> ]>

# Test quote from stdin 
xecho <[ <foo>bar</foo> ]> | xquote >{c}
xtype $c

