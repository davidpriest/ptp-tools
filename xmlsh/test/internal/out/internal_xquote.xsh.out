element()
xs:string
<test><foo>bar</foo></test>
<test>&lt;foo&gt;bar&lt;/foo&gt;&#xD;
</test>
xs:string
